src/components/SliderBox/components/SliderCard/components/SliderCardFooter/SliderCardFooter.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Button, CardActions } from '@mui/material';
+import { Link } from 'react-router-dom';
```

Comment:
`Link` should be above components from mui

Reply:
fixed